cyral/resource_cyral_repository_network_access_policy.go

```diff
@@ -149,7 +149,13 @@ func deleteRepositoryNetworkAccessPolicy() ResourceOperationConfig {
 
 func resourceRepositoryNetworkAccessPolicy() *schema.Resource {
 	return &schema.Resource{
-		Description:   "Manages the network access policy of a repository. Network access policies are also known as the [Network Shield](https://cyral.com/docs/manage-repositories/network-shield/). This feature is supported for the following repository types:" + supportedTypesMarkdown(repositoryTypesNetworkShield()),
+		Description: "Manages the network access policy of a repository. Network access policies are" +
+			" also known as the [Network Shield](https://cyral.com/docs/manage-repositories/network-shield/)." +
+			" This feature is supported for the following repository types:" +
+			supportedTypesMarkdown(repositoryTypesNetworkShield()) +
+			"\n\n-> **Note** If you also use the resource `cyral_repository_conf_auth` for the same repository," +
```

Comment:
Isn't using cyral_repository_conf_auth mandatory for any repo - I mean I think the repo will not even work unless this resource is created, right?

If so, we can remove the condition "If you also use...".

Reply:
It is not mandatory. You can create a `cyral_repository` and not provide the `cyral_repository_conf_auth` in Terraform.